packages/azure_logs/manifest.yml

```
@@ -93,7 +93,15 @@ policy_templates:
           The ingest pipeline ID to use for processing the data. If provided,
           replaces the default pipeline for this integration.
         required: false
-        show_user: true        
+        show_user: true
+      - name: storage_account_container
+        type: text
+        title: Storage Account Container
+        multi: false
+        required: false
+        show_user: false
+        description: >
+          The storage account container where the integration stores the checkpoint data for the consumer group. It is an advanced option to use with extreme care. If you are using the [processor_version](https://www.elastic.co/guide/en/integrations/current/azure_logs.html#azure_logs-event-hub-processor-v2-only) v1, you MUST use a dedicated storage account container for each event hub. See [Container Names](https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names) for details on naming rules from Microsoft. The integration generates a default container name if not specified.
```

Shouldn't this be v2 instead of v1?
```suggestion
          The storage account container where the integration stores the checkpoint data for the consumer group. It is an advanced option to use with extreme care. If you are using the [processor_version](https://www.elastic.co/guide/en/integrations/current/azure_logs.html#azure_logs-event-hub-processor-v2-only) v2, you MUST use a dedicated storage account container for each event hub. See [Container Names](https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names) for details on naming rules from Microsoft. The integration generates a default container name if not specified.
```

---

No, this limit applies to processor v1 only. 

The processor v1 stores the blobs in the container root only. So, if you want to ingest logs from the same event hub with two consumer groups (for different use cases), you must use two containers.

The processor v2 stores the blobs in folder with the `/namespace/eventhub/consumer-group/` structure, so it can use the same container.

I'll specify that processor v1 needs a dedicated container for each unique eventhub/consumer group combination, instead of event hub only. WDYT?

---

I posted an updated version of storage_account_container description.